orc8r/tools/ansible/roles/pkgrepo/tasks/main.yml

```diff
@@ -15,6 +15,7 @@
   ansible.builtin.file:
     path: /etc/apt/sources.list.d/*.list
     state: absent
+  when: preburn
 
 - name: Wait for APT Lock
```

Comment:
Not 100% sure, but I imagine this was also just necessary because of the unattended upgrades which is now gone because of the `update_ubuntu.sh` from `bento` that we use. We could try removing this, too.

Reply:
Yes, this is probably also irrelevant since the following step is executed in preburn.